assets/wizards/seo/views/tools/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment, RawHTML } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { TextControl, withWizardScreen } from '../../../../components/src';
+
+/**
+ * SEO Tools screen.
+ */
+class Tools extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		const { data, onChange } = this.props;
+		const { verification } = data;
+		const { google, bing } = verification;
+		return (
+			<Fragment>
+				<h2>{ __( 'Webmaster tools verification', 'newspack' ) }</h2>
+				<p>
+					{ __(
+						'You can use the boxes below to verify with the different Webmaster Tools. Thius feature will add a verification meta tag on your home page. Follow the links to the different Webmaster Tools and look for instructions for the meta tag verification method to get the verification code.',
```

Comment:
This description isn't super clear to me, especially around why I would want to verify with the tools or what even they are. Also you have `Thius`.

Reply:
Before we dig in on the description, is this feature even necessary, or does Site Kit take care of it (at least the Google side of things)?

Reply:
I did fix the misspelling, though, in https://github.com/Automattic/newspack-plugin/pull/435/commits/632767cb8026460be32c62bc07a5033b4a065432

Reply:
Yeah we can def. tweak further if it turns out that screen isn't super useful. That's not a blocker for this PR, though.